dashboard/config/scripts/unique_level_name_here.external

```diff
@@ -1,10 +0,0 @@
-name 'unique level name here'
```

Comment:
is removal of this file intentional?

Reply:
ugh - no.  I thought I made it, but apparently it was there before I started mucking around.  Why do we keep these bare minimum scripts?

Reply:
> Why do we keep these bare minimum scripts?

Probably because there is no way to delete scripts via levelbuilder 🙊

Reply:
oh wait, this is a level, not a script. no excuse, then